docs/replica-sets.md

```diff
@@ -98,6 +98,47 @@ unknown VirtualMachine states and graceful deletes, it might decide to already
 create new replicas in advance, to make sure that the amount of ready replicas
 stays close to the expected replica count.
 
+### Chosen Implementation
```

Comment:
Naming wise, this does not behave like a ReplicaSet.  I would recommend that you spend some time looking for an alternate naming, because you provide guarantees that far exceed a Kubernetes ReplicaSet.  It will be likely confusing for users and admins: "A VMRS is like a ReplicaSet for pods except it's kind of like a StatefulSet except when it isn't".